FRW-1442 Migration guide for Country, Currencty, Locale modules (#1563)
spryker/spryker-docs: _includes/pbc/all/upgrade-modules/upgrade-the-currency-module.md

This document describes how to upgrade the Currency module.

## Upgrading from version 3.* to version 4.0.0

In this new version of the `Currency` module, we have added support configuration currency for each store in database.

> With the `Currency` module version 4 we have added the `spy_currency_store` database table to persist stores-locales in Zed.

Reviewer: ```suggestion
With the new 4.0.0 version of the `Currency` module, we have added the `spy_currency_store` database table to persist stores-locales in Zed.
```

> Also added column `fk_currency` into  `spy_store` for save default currency per store.

Reviewer: ```suggestion
We also added the column `fk_currency` into  `spy_store`, to allow saving a default currency per store.
```

You can find more details about the changes on the [Currency module](https://github.com/spryker/currency/releases) release page.

> *Estimated migration time: 5 min*

Reviewer: ```suggestion
*Estimated migration time: 5 minutes*
```
